after/ftplugin/markdown/instant-markdown.vim

```diff
@@ -17,14 +17,44 @@ function! s:system(cmd, stdin)
     endif
 endfu
 
+" Wrapper function to automatically execute the command asynchronously and
+" redirect output in a cross-platform way. Note that stdin must be passed as a
+" List of lines.
+function! s:systemasync(cmd, stdinLines)
+    if has('win32') || has('win64')
+        call s:winasync(a:cmd, a:stdinLines)
+    else
+        let cmd = a:cmd . '&>/dev/null &'
+        call s:system(cmd, join(a:stdinLines, "\n"))
+    endif
+endfu
+
+" Executes a system command asynchronously on Windows. The List stdinLines will
+" be concatenated and passed as stdin to the command. If the List is empty,
+" stdin will also be empty.
+function! s:winasync(cmd, stdinLines)
+    " To execute a command asynchronously on windows, the script must use the
+    " "!start" command. However, stdin can't be passed to this command like
+    " system(). Instead, the lines are saved to a file and then piped into the
+    " command.
+    if len(a:stdinLines)
+        let tmpfile = tempname()
+        call writefile(a:stdinLines, tmpfile)
+        let command = 'type ' . tmpfile . ' | ' . a:cmd
+    else
+        let command = a:cmd
+    endif
+    exec 'silent !start /b cmd /c ' . command . ' > NUL'
```

Comment:
Ah - is it cause the ordering and where the pipe is is different when using the `!start` command in Windows?